src/index.js

```diff
@@ -1 +1,50 @@
+/* eslint-disable no-console */
 'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const zlip = require('zlib');
```

Comment:
There's a typo in the require statement for 'zlib'. It should be 'zlib' not 'zlip'.